### ISSUE TITLE
MDBenchmark plotting does not work with matplotlib 3.7.3

### ISSUE THREAD

**Issue by balazsfabian** [bio-phys/MDBenchmark#194]
**Expected behavior**
No error messages when plotting using.
```
>>> mdbenchmark plot --csv my_benchmark_results.csv --output-format pdf --no-gpu
Plotting GPU data only.
Plotting all hosts in input file.
Plotting all modules in your input data.
The plot was saved as 'my_benchmark_results.pdf'.
```

**Actual behavior**
```
mdbenchmark plot --csv my_benchmark_results.csv --output-format pdf --no-gpu
Plotting CPU data only.
Plotting all hosts in input file.
Plotting all modules in your input data.
Traceback (most recent call last):
  File "/u/bafabian/conda-envs/mdbtest/bin/mdbenchmark", line 8, in <module>
    sys.exit(cli())
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/mdbenchmark/cli/commands.py", line 353, in plot
    do_plot(
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/mdbenchmark/cli/plot.py", line 292, in do_plot
    fig.savefig(
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/matplotlib/figure.py", line 3328, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2362, in print_figure
    result = print_method(
  File "/u/bafabian/conda-envs/mdbtest/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2228, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
TypeError: FigureCanvasPdf.print_pdf() got an unexpected keyword argument 'type'
```
The error happens with matplotlib-3.7.0, but not with matplotlib-3.5.2

**Your version of MDBenchmark:**
Tested #192, that is
```
>>> mdbenchmark --version
mdbenchmark, version 3.0.1
```

**Comment by ksebaz** [comment]
Thanks for the report!

**Comment by ksebaz** [comment]
Fixed in #195.
